metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -498,7 +498,18 @@ func (sink *StackdriverSink) TranslateMetric(timestamp time.Time, labels map[str
 		return ts
 	case "memory/bytes_used":
 		point := sink.intPoint(timestamp, timestamp, value.IntValue)
-		return createTimeSeries(resourceLabels, memoryBytesUsedMD, point)
+		ts := createTimeSeries(resourceLabels, memoryBytesUsedMD, point)
+		ts.Metric.Labels = map[string]string{
+			"memory_type": "evictable",
+		}
+		return ts
+	case core.MetricMemoryWorkingSet.MetricDescriptor.Name:
```

Comment:
For my own education - where does this name come from?

Reply:
core.MetricMemoryWorkingSet is a metric that we fetch from kubelet. At the same time bytes_used (and minor_page_faults) are created manually in preprocessMemoryMetrics function.